src/headers/languages/c.rs

```diff
@@ -10,12 +10,13 @@ impl HeaderLanguage for C {
         self: &'_ Self,
         ctx: &'_ mut dyn Definer,
         docs: Docs<'_>,
+        deprecated: Deprecated,
         indent: &'_ Indentation,
     ) -> io::Result<()>
     {
         mk_out!(indent, ctx.out());
 
-        if docs.is_empty() {
+        if docs.is_empty() && deprecated.is_none() {
```

Comment:
this isn't exactly right, but I think getting it exactly right requires a bit of a deeper refactor.